### DISCUSSION TITLE
Missing spaces on front page of website

### DISCUSSION THREAD

**Discussion by maiki** [evennia/evennia#2607] [Evennia Engine Development]
Hiya! :wave: 

I read at [https://www.evennia.com/devblog/2021.html#2021-11-18-the-blog-moved!](https://www.evennia.com/devblog/2021.html#2021-11-18-the-blog-moved!):

> I already use github pages for the Evennia homepage and documentation. These are simply html pages located in a gh-pages branch of the main evennia repo. So I decided I would just post my blog posts in a folder and then run a markdown parser to turn it into publisheable HTML pages.

There are missing spaces in several words (likely from the recent switch over, I'm guessing, linebreaks and all), and I'm happy to go through and check all the words and submit a patch, but I can't find the source or raw HTML to edit; I checked `gh-pages` and used the search feature but did not find it.

In case this is complicated, the ones I noticed are emboldened:

>  Evennia is an open-source Python-based framework, codebase **andserver** for creating text-based multiplayer online games (aka MUD/MU* etc) using **moderntechnologies** and tools.

>  https://demo.evennia.com hosts **analmost** default Evennia server with some extra examples. Use it to get a taste **forwhat**'s possible. You can also send questions to our support channel **directlyfrom** inside the demo - just write public Hello! to say hi!

> To learn how to get your hands on the code base, the Getting Started page is the way to go. Otherwise you could browse the Documentation or **whynot** come join one of the Evennia's community's chat channels or forums. Welcome! 

Let me know where those live and I'll go through the whole doc. ^_^

**Comment by kovitikus** [comment]
lol at the 2nd one.

Anyways, you can check the Issues to see if there's documentation issues already filed for these.

Also, a lot of this stuff is fixed or will be fixed in the next major patch (1.0). So you may want to check dev branch to see if Griatch already fixed it. Some of these docs were rewritten already I think.

Otherwise, you can also open an Issue to make sure it's tracked and gets fixed.

You can link this thread in the issue to help keep tabs on things.

**Comment by Griatch** [comment]
@maiki Thanks, those need to be fixed for sure :D The front page is actually the only thing not a markdown file and lives here: https://github.com/evennia/evennia/blob/gh-pages/index.html

If you want to make a PR, feel free, otherwise it's worth a lot to just point out the errors. :)

**Comment by Griatch** [comment]
@maiki I have now updated the index page, fixing the words that were missing spaces. Check it out and thanks for the pointers!

**Reply by maiki** [reply]
Good job, and thanks for pointing out where that file is. It looks fine to me, and I'll know where to submit a patch if I notice anything else. ^_^
